clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -298,10 +303,13 @@ private void autoCommitSyncNowWithRetries(OffsetCommitRequestState requestAttemp
             if (error == null) {
                 result.complete(null);
             } else {
-                if (error instanceof RetriableException || isStaleEpochErrorAndValidEpochAvailable(error)) {
+                if ((error instanceof RetriableException || isStaleEpochErrorAndValidEpochAvailable(error))) {
```

Comment:
do we need these extra parenthesis here?

Reply:
this is failing the build actually

Reply:
Good catch!